web/html/src/manager/virtualization/nets/properties/fields-data.js

```diff
@@ -0,0 +1,55 @@
+import * as data from 'utils/data';
+
+export const mapping = {
+    bridge: {
+      description: 'Use an existing and configured bridge device. ' +
```

Comment:
I'm guessing all of these should probably be wrapped in `t()`?

Reply:
The `t()` call is where that value is used: 
```
hint={t(FieldsData.getValue(model.type, 'description', ''))}
```
But may be it would be better for automatic strings gathering to move it here indeed.